pyproject.toml

```diff
@@ -35,7 +35,7 @@ markers = "fuzzing: Run Hypothesis fuzz test suite"
 line_length = 100
 force_grid_wrap = 0
 include_trailing_comma = true
-known_third_party = ["hexbytes", "pydantic", "pytest", "setuptools"]
+known_third_party = ["eth_utils", "hexbytes", "pydantic", "pytest", "setuptools"]
```

Comment:
You can delete these and remove `seed-isort` from `pre-commit`.

It interferes with `isort` and slows it down. it is deprecated and should no longer be used.

See here: https://github.com/asottile-archive/seed-isort-config

Reply:
Removed isort [0c655d4](https://github.com/ApeWorX/evm-trace/pull/4/commits/0c655d459bf05a45bb2c97366acc8df7f940bd39)

Reply:
Removed isort [0c655d4](https://github.com/ApeWorX/evm-trace/pull/4/commits/0c655d459bf05a45bb2c97366acc8df7f940bd39)